mutable_tree.go

```diff
@@ -45,6 +46,18 @@ func (tree *MutableTree) VersionExists(version int64) bool {
 	return tree.versions[version]
 }
 
+// AvailableVersions returns all available versions
+func (tree *MutableTree) AvailableVersions() []int {
+	res := make([]int, 0, len(tree.versions))
+	for i, v := range tree.versions {
+		if v {
+			res = append(res, int(i))
+		}
+	}
+	sort.Sort(sort.IntSlice(res))
```

Comment:
Shouldn't one sort be enough here?

Reply:
There is only one sort.
`sort.IntSlice` is just a [type to enable sorting []int](https://godoc.org/sort#IntSlice)
`sort.Sort` actually does the work.

Reply:
Thanks for the clarification!